BookReader/Classes/BookViewController.swift

```diff
@@ -88,7 +88,7 @@ public class BookViewController: UIViewController, UIPopoverPresentationControll
         NotificationCenter.default.addObserver(self, selector: #selector(pdfViewPageChanged(_:)), name: .PDFViewPageChanged, object: nil)
 
         self.barHideOnTapGestureRecognizer.addTarget(self, action: #selector(gestureRecognizedToggleVisibility(_:)))
```

Comment:
This line also needs to be removed since you duplicated it on line 101.

Reply:
Woops! I thought I had removed it.